src/main/java/school/faang/Main.java

```diff
@@ -1,4 +1,25 @@
 package school.faang;
 
 public class Main {
+    public static void main(String[] args) {
+        checkUser("Alex", 20, "Google", "London");
+        checkUser("", 20, "Google", "London");
+        checkUser(null, 20, "Google", "London");
+        checkUser("Alex", 10, "Google", "London");
+        checkUser("Alex", 20, "Burger King", "London");
+        checkUser("Alex", 20, "Google", "USA");
+
+    }
+
+    public static void checkUser(String name, int age, String job, String address) {
+        try {
+            User user = new User(name, age, job, address);
+            System.out.printf("Пользователь (%s, %d, %s, %s) был успешно создан\n",
```

Comment:
Сообщения в консоль обычно пишутся через инструменты для логирования (logger). Он уже вшит в ломбок, можно навесить аннотацию @Slf4j над классом, тогда sout можно будет заменить на 
```suggestion
            log.info("Пользователь {} был успешно создан", user);
```